Jenkinsfile

```diff
@@ -48,14 +47,11 @@ elifeLibrary {
             }
         }
 
-        elifeMainlineOnly {
+        elifeTagOnly { tag ->
             stage 'Push release', {
-                isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
-                if (isNew) {
-                    sh "IMAGE_TAG=${commit} " +
-                        "docker-compose -f docker-compose.yml -f docker-compose.ci.yml run " +
-                        "sciencebeam-utils-py2 twine upload dist/*"
-                }
+                sh "IMAGE_TAG=${commit} " +
```

Comment:
doesn't seem to need a version input, because it's still stored in the code

Reply:
should we maybe generate a versions file that is read by `__init__.py`?

Reply:
Seems to still potentially contradict https://github.com/elifesciences/tech-team/pull/17 but it's reasonable for an application to know its own version number in some way. What it is used for?

Reply:
It's an optional convention to provide the version number, specified by [PEP 396](https://www.python.org/dev/peps/pep-0396/).
I think it's still in compliance with the ADR if the version number is derived from the tag. It doesn't necessarily need to appear in GitHub, as long as it's part of the release published to pypi.

Reply:
Seems like the best of both worlds, no forgotten version numbers committed in `master` and the capability to introspect the version from Python when installing the library.
How do we add it? Seems part of the `Dockerfile` with a build argument being passed?

Reply:
Implemented with a `sed` in the `Dockerfile`.